Makefile

```diff
@@ -22,3 +22,12 @@ serve-skore-ui:
 		--host 0.0.0.0 \
 		--port 22140 \
 		--timeout-graceful-shutdown 0
+
+test-frontend:
+	cd skore-ui && npm install
+	cd skore-ui && npm run test:unit
+
+test-backend:
+	cd skore && pytest tests
```

Comment:
```suggestion
	cd skore && python -m pytest --cache-clear --doctest-modules src/ tests/
```

Reply:
This command outputs warning which could be puzzling...

> Users/rouk1/dev/skore/venv/lib/python3.12/site-packages/coverage/inorout.py:508: CoverageWarning:
> Module src/ was never imported. (module-not-imported)
> /Users/rouk1/dev/skore/venv/lib/python3.12/site-packages/coverage/inorout.py:508: CoverageWarning:
> Module tests/ was never imported. (module-not-imported)
> /Users/rouk1/dev/skore/venv/lib/python3.12/site-packages/coverage/control.py:892: CoverageWarning:
> No data was collected. (no-data-collected)
> 
> WARNING: Failed to generate report: No data to report.

Reply:
Solution: don't compute coverage (add `--no-cov`)

Reply:
Or cd to the correct folder and get coverage + color output :)

Reply:
Please can you update to include cache cleaning/doctest as proposed (with you cd)?